src/lambda_vm/checks.ml

```diff
@@ -0,0 +1,8 @@
+exception Out_of_stack
+exception Out_of_gas
+
+(* TODO: reasonable number here *)
+let max_stack_depth = 1000
```

Comment:
What could be `reasonable number`?
Is there any way to calculate it?

Reply:
I updated this with the explanation, but there is still some work to be done verifying my hypothesis, because of that I was very conservative.